formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
> t's unfortunate gocc doesn't give greater flexibility 

If you figure out a clean and generic approach we may add it to gocc. How did it work in ragel? The Gocc team is really lovely, and they often have a quick response time to new design ideas and PRs.